### DISCUSSION TITLE
TLS Version

### DISCUSSION THREAD

**Discussion by jsherne** [provectus/kafka-ui#2458] [Q&A] [answered]
Please could someone tell me which version of TLS is implemented at Ingress level for Kafka UI?

Many thanks

**Comment by Haarolean** [comment, marked as answer]
If you enable SSL via `server.ssl.enabled` property, you can set the TLS version via these properties:
```
server.ssl.protocol=TLS
server.ssl.enabled-protocols=TLSv1.2
```
